.github/workflows/app.yml

```diff
@@ -1,12 +1,26 @@
 name: "frontend / app"
 on:
   pull_request:
-    paths-ignore:
-      - 'sdks/**'
 
 jobs:
+  check_path:
+    runs-on: ubuntu-latest
+    outputs:
+      sdk_only: ${{ steps.filter.outputs.sdk_only }}
+    steps:
+      - uses: actions/checkout@v4
+      - uses: dorny/paths-filter@v2
```

Comment:
Is there no way we can just use a simple bash script here instead? Given recent security incidents with third-party GHA scripts ([example](https://orca.security/resources/blog/github-action-tj-actions-changed-files-compromised)) I'd prefer to avoid relying on this